VTEX - Master Data API - v2.json

```diff
@@ -1193,6 +1193,362 @@
+                "tags": [
+                    "Customer profiles"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/components/schemas/ShopperSchemaResponse"
+                                    }
+                                },
+                                "example": [
+                                    {
+                                        "name": "v1",
+                                        "schema": {
+                                            "title": "shopper",
+                                            "type": "object",
+                                            "properties": {
```

Comment:
The schema is incorrect. The issue described in this [comment](https://github.com/vtex/openapi-schemas/pull/1556#discussion_r2561937087) remains.